modern/src/settings/ComputedAttributesPage.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import MainToolbar from '../MainToolbar';
+import { TableContainer, Table, TableRow, TableCell, TableHead, TableBody, makeStyles, IconButton } from '@material-ui/core';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import t from '../common/localization';
+import { useEffectAsync } from '../reactHelper';
+import EditCollectionView from '../EditCollectionView';
+
+const useStyles = makeStyles(theme => ({
+  columnAction: {
+    width: theme.spacing(1),
+    padding: theme.spacing(0, 1),
+  },
+}));
+
+const ComputedAttributeView = ({ updateTimestamp, onMenuClick }) => {
+  const classes = useStyles();
+
+  const [items, setItems] = useState([]);
+
+  useEffectAsync(async () => {
+    const response = await fetch('/api/attributes/computed');
+    if (response.ok) {
+      setItems(await response.json());
+    }
+  }, [updateTimestamp]);
+
+  return (
+    <TableContainer>
+    <Table>
+      <TableHead>
+        <TableRow>
+          <TableCell className={classes.columnAction} />
+          <TableCell>{t('sharedDescription')}</TableCell>
+          <TableCell>{t('sharedAttribute')}</TableCell>
+          <TableCell>{t('sharedExpression')}</TableCell>
+          <TableCell>{t('sharedType')}</TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {items.map((item) => (
+          <TableRow key={item.id}>
+            <TableCell className={classes.columnAction} padding="none">
```

Comment:
We only allow admins to create/edit attributes. Can we hide this and add button if the user is not an admin?